sklearn/model_selection/tests/test_split.py

```diff
@@ -562,7 +563,7 @@ def test_stratified_shuffle_split_even():
     # Test the StratifiedShuffleSplit, indices are drawn with a
     # equal chance
     n_folds = 5
-    n_iter = 1000
+    n_splits = 1000
```

Comment:
After modified [this line](https://github.com/scikit-learn/scikit-learn/pull/7187/files#diff-3f55cc702cd72cfa9923bf58091924f3R587), I guess `n_splits` can remain the same name?